docs/rfcs/0000-namespace.md

```diff
@@ -0,0 +1,76 @@
+- Start Date: (2015-10-29)
+- RFC PR: (leave this empty)
+- AMS Issue: (leave this empty)
+
+# Summary
+
+Provide a consistent API for the user of the AMS.
+
+# Motivation
+
+The actual public API is defined under `ActiveModelSerializers`,
+`ActiveModel::Serializer` and `ActiveModel`.
```

Comment:
|Current | Desired | Notes |
|-----------|-------------|---------|
|`ActiveModelSerializers` and `ActiveModel::Serializer` | `ActiveModel::Serializers` | The main namespace |
| `ActiveModelSerializers.logger` | `ActiveModel::Serializers.logger` | |
|`ActiveModelSerializers::Model`| `ActiveModel::Serializers::Model` ||
|`ActiveModel::SerializableResource`| `ActiveModel::Serializers::SerializableResource` | |
| `ActiveModel::Serializer` | `ActiveModel::Serializers::Serializer` | I know that is probably a bad name, but In a second moment we can rename this to `Resource` [for example following this idea](https://github.com/rails-api/active_model_serializers/pull/1301/files#r42963185)|
|`ActiveModel::Serializer.config`| `ActiveModel::Serializers.config` ||

This way we will have a single namespace to keep all the AMS into a single namespace `ActiveModel::Serializers`.

It seens to me that the actual architeture leads to some confusion to the development team as you can see [on this comment](https://github.com/rails-api/active_model_serializers/pull/1313#discussion_r43573370) this is not clear where to put a new implementation, I feel it too when I am working on the test helpers. 

I think if we have a single namespace this will be more clear to the team and to the end users too.